[2024-08-06 21:35:50,630: INFO: 2114832597: >>>>>> Data Ingestion Stage started <<<<<<]
[2024-08-06 21:35:50,632: ERROR: 2114832597: [Errno 2] No such file or directory: 'config\\config.yaml']
Traceback (most recent call last):
  File "C:\Users\admin\AppData\Local\Temp\ipykernel_15368\2114832597.py", line 20, in <module>
    obj.main()
  File "C:\Users\admin\AppData\Local\Temp\ipykernel_15368\2114832597.py", line 9, in main
    config = ConfigurationManager()
  File "e:\mlops\kidney_app\src\kidneyDiseaseClassifier\config\configuration.py", line 26, in __init__
    self.config = read_yaml(config_filepath)
  File "c:\Users\admin\anaconda3\envs\kidney\lib\site-packages\ensure\main.py", line 849, in __call__
    return_val = self.f(*args, **kwargs)
  File "e:\mlops\kidney_app\src\kidneyDiseaseClassifier\utils\common.py", line 36, in read_yaml
    raise e
  File "e:\mlops\kidney_app\src\kidneyDiseaseClassifier\utils\common.py", line 29, in read_yaml
    with open(path_to_yaml) as yaml_file:
FileNotFoundError: [Errno 2] No such file or directory: 'config\\config.yaml'
[2024-08-06 21:40:45,418: INFO: 1737822578: >>>>>> Data Ingestion Stage started <<<<<<]
[2024-08-06 21:40:45,419: ERROR: 1737822578: [Errno 2] No such file or directory: 'config\\config.yaml']
Traceback (most recent call last):
  File "C:\Users\admin\AppData\Local\Temp\ipykernel_15368\1737822578.py", line 20, in <module>
    obj.main()
  File "C:\Users\admin\AppData\Local\Temp\ipykernel_15368\1737822578.py", line 9, in main
    config = ConfigurationManager()
  File "e:\mlops\kidney_app\src\kidneyDiseaseClassifier\config\configuration.py", line 26, in __init__
    self.config = read_yaml(config_filepath)
  File "c:\Users\admin\anaconda3\envs\kidney\lib\site-packages\ensure\main.py", line 849, in __call__
    return_val = self.f(*args, **kwargs)
  File "e:\mlops\kidney_app\src\kidneyDiseaseClassifier\utils\common.py", line 36, in read_yaml
    raise e
  File "e:\mlops\kidney_app\src\kidneyDiseaseClassifier\utils\common.py", line 29, in read_yaml
    with open(path_to_yaml) as yaml_file:
FileNotFoundError: [Errno 2] No such file or directory: 'config\\config.yaml'
[2024-08-06 21:41:07,627: INFO: 1737822578: >>>>>> Data Ingestion Stage started <<<<<<]
[2024-08-06 21:41:07,630: ERROR: 1737822578: [Errno 2] No such file or directory: 'config\\config.yaml']
Traceback (most recent call last):
  File "C:\Users\admin\AppData\Local\Temp\ipykernel_19452\1737822578.py", line 20, in <module>
    obj.main()
  File "C:\Users\admin\AppData\Local\Temp\ipykernel_19452\1737822578.py", line 9, in main
    config = ConfigurationManager()
  File "e:\mlops\kidney_app\src\kidneyDiseaseClassifier\config\configuration.py", line 26, in __init__
    self.config = read_yaml(config_filepath)
  File "c:\Users\admin\anaconda3\envs\kidney\lib\site-packages\ensure\main.py", line 849, in __call__
    return_val = self.f(*args, **kwargs)
  File "e:\mlops\kidney_app\src\kidneyDiseaseClassifier\utils\common.py", line 36, in read_yaml
    raise e
  File "e:\mlops\kidney_app\src\kidneyDiseaseClassifier\utils\common.py", line 29, in read_yaml
    with open(path_to_yaml) as yaml_file:
FileNotFoundError: [Errno 2] No such file or directory: 'config\\config.yaml'
[2024-08-06 22:16:32,890: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-08-06 22:16:32,896: INFO: common: yaml file: params.yaml loaded successfully]
[2024-08-06 22:16:32,899: INFO: common: created directory at: artifacts]
[2024-08-06 22:16:32,901: INFO: common: created directory at: artifacts/data_ingestion]
[2024-08-06 22:16:34,047: INFO: 2449113567: >>>>>> Data Ingestion Stage started <<<<<<]
[2024-08-06 22:16:34,053: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-08-06 22:16:34,058: INFO: common: yaml file: params.yaml loaded successfully]
[2024-08-06 22:16:34,060: INFO: common: created directory at: artifacts]
[2024-08-06 22:16:34,062: INFO: common: created directory at: artifacts/data_ingestion]
[2024-08-06 22:16:34,064: INFO: 2449113567: >>>>>> Data Ingestion Stage completed <<<<<<<

x========================x]
